jobbergate-cli/CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@ This file keeps track of all notable changes to jobbergate-cli
 
 Unreleased
 ----------
+* Complete re-write of the Jobbergate CLI
```

Comment:
```suggestion
 * Complete re-write of the Jobbergate CLI
```

Reply:
changed bullet list format to match rest of document in latest commit.